packages/playground/src/router/index.ts

```diff
@@ -469,7 +469,7 @@ function createDeployRoutes(): RouteRecordRaw[] {
       children: [
         {
           path: DashboardRoutes.Deploy.PricingCalculator,
-          component: () => import("../calculator/resource_pricing.vue"),
+          component: () => import("../calculator/PricingCalculator.vue"),
```

Comment:
i don't think we rename files with camelcase

Reply:
Updated